### ISSUE TITLE
two drop down menu validation

### ISSUE THREAD

**Issue by geethupendra** [posabsolute/jQuery-Validation-Engine#449]
I have corded 2 drop down menu ..

id="pay_from"
id="pay_to"

<select name="pay_from" class="validate[required]" id="pay_from" >
                        <option value="" selected="selected">Select Month</option>
                        <option value="<?php echo date('Y')?>-01-01"><?php echo date('Y')?>-Jan</option>
                        <option value="<?php echo date('Y')?>-02-01"><?php echo date('Y')?>-Feb</option>
                        <option value="<?php echo date('Y')?>-03-01"><?php echo date('Y')?>-Mar</option>
                        <option value="<?php echo date('Y')?>-04-01"><?php echo date('Y')?>-Apr</option>
                        <option value="<?php echo date('Y')?>-05-01"><?php echo date('Y')?>-May</option>
                        <option value="<?php echo date('Y')?>-06-01"><?php echo date('Y')?>-Jun</option>
                        <option value="<?php echo date('Y')?>-07-01"><?php echo date('Y')?>-Jul</option>
                        <option value="<?php echo date('Y')?>-08-01"><?php echo date('Y')?>-Aug</option>
                        <option value="<?php echo date('Y')?>-09-01"><?php echo date('Y')?>-Sep</option>
                        <option value="<?php echo date('Y')?>-10-01"><?php echo date('Y')?>-Oct</option>
                        <option value="<?php echo date('Y')?>-11-01"><?php echo date('Y')?>-Nov</option>
                        <option value="<?php echo date('Y')?>-12-01"><?php echo date('Y')?>-Dec</option>
                      </select>

Basically i need "pay_to" grater than "pay_from" validation,...if we selected from (2012-MAY) it should be validate (2012-MAY or JUN / JUL / ..etc...).., Please kindly help to sort this..

thanks
geeth
